### ISSUE TITLE
Prop/DCDC power loss

### ISSUE THREAD

**Issue by 18kudlugim** [pathfinder-for-autonomous-navigation/FlightSoftware#591]
We need a PTest case that tests the interaction between DCDC and the Prop System. The PropController can only change state if the DCDC SpikeDock pin is set to HIGH. We want to ensure that this behavior is stable.

**Comment by shihaocao** [comment]
#609 Does this change things?

**Comment by athena255** [comment]
Yes, Prop now sets DCDC pin to HIGH before it attempts to open valves. The ptest case should check for instances when Prop has already set the DCDC pin to HIGH, but test behavior where the pin is suddenly LOW. The states where this matters is: Await_Pressurizing, Await_Firing, Firing, and Venting. 

A unit test has been written to check this behavior when the Prop is in Await_Pressurizing. See `test_prop_controller.cpp`: test_not_enough_cycles_for_dcdc_high, test_enough_cycles_for_dcdc_high, test_require_consecutive_dcdc_high.

**Comment by shihaocao** [comment]
@18kudlugim is this cleared per latest batch of PRs?
